src/index.js

```diff
@@ -154,18 +158,19 @@ const render = async (data) => {
     label.innerHTML = likeTemplate;
 
     const noLikes = document.createElement('span');
-    noLikes.classList.add('no-likes');
+    // noLikes.classList.add('no-likes');
     noLikes.classList.add(`a${item.idMeal}`);
-    noLikes.textContent = '';
+
+    noLikes.textContent = `${likesData[item.idMeal]?.likes || 0} likes`;
 
     const btn = document.createElement('button');
     btn.classList.add('explore');
     btn.textContent = 'View Recipe';
-     btn.addEventListener('click', async () => {
-      const cardId = btn.parentElement.parentElement.dataset.set;
+    btn.addEventListener('click', async () => {
+      const cardId = btn.parentElement.parentElement.parentElement.dataset.set;
       displayPopup(item, cardId);
     });
-    
+
     info.appendChild(title);
     likeContainer.appendChild(likeInput);
     likeContainer.appendChild(label);
```

Comment:
I suggest you use the append method if you have more than one child instead of the appendChild method